cmd/src/cli/run.rs

```diff
@@ -1,39 +1,58 @@
 use cli::{self, package};
+use colored::*;
 use error::DefaultResult;
 use holochain_container_api::{config::*, container::Container};
+use std::fs;
+
+const LOCAL_STORAGE_PATH: &str = ".hc";
+
+const AGENT_CONFIG_ID: &str = "hc-run-agent";
+const DNA_CONFIG_ID: &str = "hc-run-dna";
+const INSTANCE_CONFIG_ID: &str = "test-instance";
+const INTERFACE_CONFIG_ID: &str = "websocket-interface";
 
 /// Starts a small container with the current application running
-pub fn run(package: bool, port: u16) -> DefaultResult<()> {
+pub fn run(package: bool, port: u16, persist: bool) -> DefaultResult<()> {
```

Comment:
persist is not a boolean though is it? in the not-so-distant-future we will have sqlite as an option too right?

Reply:
Fur the purposes of dev and `hc run` I think on option is fine for now...

Reply:
@zippy would it be OK to swap it with an enum that implements `From<String>` so we can extend it easily as we go? 

now is the easiest moment to make that refactor i think

Reply:
Good idea but this seems like a follow-up though